public/legacy/modules/AOS_Products/AOS_Products.php

```diff
@@ -64,10 +64,12 @@ public function save($check_notify = false)
         require_once('include/upload_file.php');
         $GLOBALS['log']->debug('UPLOADING PRODUCT IMAGE');
 
+        if(!empty($_FILES['uploadimage'])){
         $imageFileName = $_FILES['uploadimage']['name'] ?? '';
```

Comment:
Hi @bralandealmeida,

Thank you for your PR.

Just a small thing, the following line needs one more indentation level

```php
$imageFileName = $_FILES['uploadimage']['name'] ?? ''; 
```
Like:

````php
if(!empty($_FILES['uploadimage'])){
    $imageFileName = $_FILES['uploadimage']['name'] ?? '';
    if (!has_valid_image_extension('AOS_Products Uploaded image file: ' . $imageFileName , $imageFileName)) {
        LoggerManager::getLogger()->fatal("AOS_Products save - Invalid image file ext : '$imageFileName'.");
         throw new RuntimeException('Invalid request');
    }
}

```

Reply:
Thanks! Fixed.